tests/Microsoft.Identity.Test.Integration.net45/Infrastructure/ClientAppTestExtensions.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Diagnostics;
+using Microsoft.Identity.Client;
+
+namespace Microsoft.Identity.Test.Integration.net45.Infrastructure
+{
+    public static class ClientAppTestExtensions
+    {
+        internal static PublicClientApplicationBuilder WithTestLogging(this PublicClientApplicationBuilder builder)
+        {
+            var httpClientFactory = new HttpSnifferClientFactory();
+
+            return builder
+                .WithLogging((lvl, msg, pii) => Trace.WriteLine($"[MSAL][{lvl}] {msg}"), LogLevel.Verbose, enablePiiLogging: true)
```

Comment:
I assume we don't have PII in tests, so default true is fine?

Reply:
Well, I log the Http request and the Http response, so there is some sensitive stuff in there, but related to some test users in the lab, so nothing impactful. 

That said, the test logs live on ADO where they are private. I did not see the test logs being shared from ADO to GitHub / public view.

Reply:
Well spotted, btw.